espnet2/asr/frontend/s3prl.py

```diff
@@ -36,75 +21,37 @@ def __init__(
         download_dir: str = None,
         multilayer_feature: bool = False,
     ):
+        try:
+            from s3prl.nn import Featurizer
+            from s3prl.nn import S3PRLUpstream
+        except Exception as e:
+            print("Error: S3PRL is not properly installed.")
+            print("Please install S3PRL: cd ${MAIN_ROOT}/tools && make s3prl.done")
+            raise e
+
         assert check_argument_types()
         super().__init__()
+
         if isinstance(fs, str):
             fs = humanfriendly.parse_size(fs)
+        if fs != 16000:
+            logging.warning("All the upstream models in S3PRL now only support 16 kHz audio.")
 
+        # TODO(simpleoier): to remove download_dir and update related configs.
         if download_dir is not None:
             torch.hub.set_dir(download_dir)
```

Comment:
Now we don't download checkpoint to the directory defined by torch.hub. We download to [./cache/s3prl/download](https://github.com/s3prl/s3prl/blob/20b0664a5f88de8a2c16c871b8e866731b7a9d19/s3prl/util/download.py#L18-L28). We also have an API (despite not yet on the documentation page, `s3prl.util.download.set_dir`) to change this directory if you wish to change it!

Reply:
Yes, I found that in the s3prl code. I will delete this and the content in line 40-42 soon. But since this is a tentative PR and many existing configs still use download_dir, it's easier to pass the test and let others test the s3prl frontend in the meantime.

Reply:
Oh, then maybe you can change `torch.hub.set_dir(download_dir)` to `s3prl.util.download.set_dir(download_dir)`?

Reply:
OK, let me change to this. Thanks!